### ISSUE TITLE
the model of Tensorflow2.0 lite has low accuracy, is there anything wrong?

### ISSUE THREAD

**Issue by btdan** [tensorflow/tensorflow#34960]
Hello, I create new model with the program as follows:
https://github.com/tensorflow/hub/blob/master/tensorflow_hub/tools/make_image_classifier/make_image_classifier.py
and after the whole train process, the accuracy is more than 93%
However, I  run the program as follows to test the model:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/examples/python/label_image.py
with the same dataset as validacation datasets. But the accuracy is very low, less than 80%.
Is there anything wrong? Thanks very much for your help.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Ubuntu 16.04
- TensorFlow installed from (source or binary):
binary
- TensorFlow version (or github SHA if from source):
2.0.0

**Command used to run the converter or code if you’re using the Python API**

make_image_classifier --image_dir /home/ioz/bird/images/ --tfhub_module /home/tensorflow-2.0/resnet101v4/ --image_size 224 --saved_model_dir /home/ioz/bird/models/ --labels_output_file /home/ioz/bird/m
odels/class_labels.txt --tflite_output_file /home/ioz/bird/models/resnet101_20_0.001True_model.tflite --learning_rate 0.001 --train_epochs 20 --do_fine_tuning True

**Comment by terryheo** [comment]
https://github.com/tensorflow/hub/tree/master/tensorflow_hub/tools/make_image_classifier shows Top-K accuracy with the given dataset. But https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/examples/python/label_image.py shows softmax output with the given single image which is not the Top-K accuracy.

**Comment by ymodak** [comment]
Closing this issue since it's resolved. Feel free to reopen if have further questions. Thanks!
